src/AccessToken.php

```diff
@@ -394,74 +393,24 @@ private function retrieveCertsFromLocation($url, array $options = [])
      */
     private function checkAndInitializePhpsec()
     {
-        if (!$this->checkAndInitializePhpsec2() && !$this->checkPhpsec3()) {
+        if (!class_exists(RSA::class)) {
             throw new RuntimeException('Please require phpseclib/phpseclib v2 or v3 to use this utility.');
         }
     }
 
     private function loadPhpsecPublicKey(string $modulus, string $exponent): string
     {
-        if (class_exists(RSA::class) && class_exists(BigInteger2::class)) {
-            $key = new RSA();
-            $key->loadKey([
-                'n' => new BigInteger2($this->callJwtStatic('urlsafeB64Decode', [
-                    $modulus,
-                ]), 256),
-                'e' => new BigInteger2($this->callJwtStatic('urlsafeB64Decode', [
-                    $exponent
-                ]), 256),
-            ]);
-            return $key->getPublicKey();
-        }
         $key = PublicKeyLoader::load([
-            'n' => new BigInteger3($this->callJwtStatic('urlsafeB64Decode', [
+            'n' => new BigInteger($this->callJwtStatic('urlsafeB64Decode', [
                 $modulus,
             ]), 256),
-            'e' => new BigInteger3($this->callJwtStatic('urlsafeB64Decode', [
+            'e' => new BigInteger($this->callJwtStatic('urlsafeB64Decode', [
                 $exponent
             ]), 256),
         ]);
         return $key->toString('PKCS8');
     }
 
-    /**
-     * @return bool
-     */
-    private function checkAndInitializePhpsec2(): bool
-    {
-        if (!class_exists('phpseclib\Crypt\RSA')) {
-            return false;
-        }
-
-        /**
-         * phpseclib calls "phpinfo" by default, which requires special
-         * whitelisting in the AppEngine VM environment. This function
-         * sets constants to bypass the need for phpseclib to check phpinfo
-         *
-         * @see phpseclib/Math/BigInteger
-         * @see https://github.com/GoogleCloudPlatform/getting-started-php/issues/85
-         * @codeCoverageIgnore
-         */
-        if (filter_var(getenv('GAE_VM'), FILTER_VALIDATE_BOOLEAN)) {
-            if (!defined('MATH_BIGINTEGER_OPENSSL_ENABLED')) {
-                define('MATH_BIGINTEGER_OPENSSL_ENABLED', true);
-            }
-            if (!defined('CRYPT_RSA_MODE')) {
-                define('CRYPT_RSA_MODE', RSA::MODE_OPENSSL);
-            }
-        }
```

Comment:
This part is no longer needed since phpseclib no longer calls `phpinfo()`, ref commit: https://github.com/phpseclib/phpseclib/commit/7a628c62b0071d03bde6be82c2f77acdadb20df2